packages/main/src/pages/static/contact-us/components/ContactSideBar.module.css

```diff
@@ -0,0 +1,56 @@
+.sidebarContainer{
+    background-color: #009967;
+    color: white;
+    padding: 40px 32px;
+}
+.topTitleContainer{
+    text-align: center;
+}
+
+.topTitleContainer h1{
+    font-size: 1.4rem;
+    font-weight: 600;
+    margin-bottom: 0;
+}
+.topTitleContainer p {
+    font-size: 13px !important;
+    margin-top: -10px;
+}
+
+.contactMenuDataContainer{
+    margin-top: 32px;
+}
+.contactDataGroup{
+    display: flex;
+    align-items: start;
+    gap: 12px;
+    margin-bottom: 2rem;
+}
+.contactDataIcon{
+    height: 15px;
+}
+.contactData{
+    margin-top: -2px;
+    display: flex;
+    flex-direction: column;
+    gap: 1;
+}
+.contactDataTitle{
+    font-weight: 500;
+}
+.contactDataDesc{
+    font-size: 90% !important;
```

Comment:
are these `!importants` necessary

Reply:
I added it because i think something else was overriding it

Reply:
Look for what's overriding the style and deal with that first then apply your styles where you need them

Reply:
Already solved it